tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -1090,6 +1093,65 @@ bool FindMatMulBiasAddAndGelu(RemapperContext* ctx, int node_index,
   return (found_gelu_exact || found_gelu_approximate);
 }
 
+bool FindSigmoidAndMul(RemapperContext* ctx, int node_index,
```

Comment:
Since this pattern is specific to oneDNN, please check at the beginning if oneDNN is enabled and return false if it's not.

Reply:
Done